CHANGELOG.md

```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
+## [5.0.0] - 2022-02-22
```

Comment:
This is because we can no longer instantiate `TokenProvider` right?

Reply:
Yes because it is `abstract`. Also, it is not exported anymore, so even the imports need to change.